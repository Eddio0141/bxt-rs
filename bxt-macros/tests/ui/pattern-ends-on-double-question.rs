use bxt_macros::pattern;

fn main() {
    pattern!(00 ??);
}
