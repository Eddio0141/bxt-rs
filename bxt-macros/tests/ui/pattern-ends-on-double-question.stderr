error: pattern ends on `??` (probably not what you want)
 --> $DIR/pattern-ends-on-double-question.rs:4:18
  |
4 |     pattern!(00 ??);
  |                  ^
